javacutil/src/main/java/org/checkerframework/javacutil/CollectionUtils.java

```diff
@@ -23,4 +28,203 @@ protected boolean removeEldestEntry(Map.Entry<K, V> entry) {
       }
     };
   }
+
+  ///
+  /// Temporary utility methods
+  ///
+
+  // TODO: After code review, copy these methods into plume-util.
+  // After plume-util 1.6.6 is released, use these methods from it.
+
+  /**
+   * Returns a copy of {@code orig}, where each element of the result is a clone of the
+   * corresponding element of {@code orig}.
+   *
+   * @param <T> the type of elements of the collection
+   * @param <C> the type of the collection
+   * @param orig a collection
+   * @return a deep copy of {@code orig}
+   */
+  @SuppressWarnings({
+    "signedness", // problem with clone()
+    "nullness:return" // "return null;"
+  })
+  public static <@Nullable T, C extends @Nullable Collection<T>> C cloneElements(C orig) {
```

Comment:
I think, because of implicit bounds defaulting (https://checkerframework.org/manual/#null-defaults), `@Nullable T` and `T` are equivalent here. `@Nullable T` is clearer, I suppose, since we really intend this to be nullable?

Reply:
Unrelated comment on the same line: should this method be `@PolyNull` (on the return type and the arguments)?

Reply:
> Unrelated comment on the same line: should this method be `@PolyNull` (on the return type and the arguments)?

I think that is not necessary because of the Java polymorphism.